Tasks/PipAuthenticateV0/task.json

```diff
@@ -64,7 +64,7 @@
             "endpointId": "tfs:feed",
             "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds",
             "resultSelector": "jsonpath:$.value[*]",
-            "resultTemplate": "{ \"Value\" : \"{{{name}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
+            "resultTemplate": "{ \"Value\" : \"{{#if project}}{{{project.name}}}\\/{{/if}}{{{name}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
```

Comment:
@aasim Note that the slash is optional and included only if there is a project.  Also, since it is usual for python to refer to the registry, it is done here by name instead of by id, necessary for backcompat

Reply:
Thanks for the update Peter, I will check and add it to nuget/dotnetcorecliv2 as well.